doc/developers/develop.rst

```diff
@@ -723,6 +723,40 @@ method taking no input and returning a boolean. If this method exists,
 See :ref:`sphx_glr_auto_examples_developing_estimators_sklearn_is_fitted.py`
 for an example on how to use the API.
 
+Developer API for HTML representation
+=====================================
+
+.. warning::
+
+    The HTML representation API is experimental and the API is subject to change.
+
+Estimators inheriting from :class:`~sklearn.base.BaseEstimator` gain a functionality
+allowing to display an HTML representation of estimators in interactive programming
+environments such as Jupyter notebooks. For instance, we can display this HTML
+diagram::
+
+    from sklearn.base import BaseEstimator
+
+    BaseEstimator()
+
+The raw HTML representation can be obtained by invoking the function
+:func:`~sklearn.utils.estimator_html_repr` on an estimator instance. This representation
+is not intended to be tweak. The only supported use case is to properly redirect the
```

Comment:
```suggestion
is not intended to be tweaked. The only supported use case is to properly redirect the
```
I am not sure what you are trying to say here. The way I read it is that after calling `estimator_html_repr` I should not modify the result, but the next sentence talks about modifying it?

Overall I am not sure what we are trying to tell the user in this paragraph.

Reply:
The meaning here is that you cannot modify the output of `estimator_html_repr` with parameters. We are not accepting any new parameter.

However, setting the parameter link to the "doc" will have an impact on the output but not through the `estimator_hmlt_repr`.